### ISSUE TITLE
Improvement to SNR condition

### ISSUE THREAD

**Issue by joakru-astro** [gautam-404/PreWhitener#1]
Another approach for deciding if peak is significant might be to consider SNR in a box of given size centred around peak's frequency.

I'm not sure if it is inherently better but might be worth testing and seems as pretty common approach.

**Comment by gautam-404** [comment]
Makes sense. Instead of a global SNR, we can compute local SNR for the peaks. Potentially will also make the code run faster.
